lib/ngx/balancer.lua

```diff
@@ -38,6 +38,10 @@ if subsystem == 'http' then
     int ngx_http_lua_ffi_balancer_set_timeouts(ngx_http_request_t *r,
         long connect_timeout, long send_timeout,
         long read_timeout, char **err);
+
+    int
+    ngx_http_lua_ffi_balancer_recreate_request(ngx_http_request_t *r,
```

Comment:
style: FFI prototype declarations should have the return type on the same line, like in the C module